### PR TITLE
fix unmodifiable Map

### PR DESCRIPTION
**Pull request by josxha**
Fixes https://github.com/ezet/stripe-sdk/issues/129

### CODE REVIEW

**Review by ezet — APPROVED** [review]

File: lib/src/stripe_api.dart [hunk 1]
```diff
@@ -91,7 +91,8 @@ class StripeApi {
 
   /// Confirm a PaymentIntent
   /// https://stripe.com/docs/api/payment_intents/confirm
-  Future<Map<String, dynamic>> confirmPaymentIntent(String clientSecret, {Map<String, dynamic> data = const {}}) async {
```
Comment (ezet) [inline lib/src/stripe_api.dart:94]: Would it be enough to just remove the `const` modifier?

**Comment by francislions** [comment]
I tried the fix editing the stripe_sdk in pub-cache and it works! 👍🏻

**Comment by ezet** [comment]
@francislions I won't release another version to pub yet, so I suggest you pull the library from git for now.

File: lib/src/stripe_api.dart (hunk 1 above)
Comment (josxha) [reply lib/src/stripe_api.dart:94]: That was what I originally wanted to write, but was not possible.
`error: The default value of an optional parameter must be constant. (non_constant_default_value at [stripe_sdk] lib\src\stripe_api.dart:113)`

**Pull request merged**